[2024-09-11T01:59:55.276+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T01:59:55.341+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.seccond_task scheduled__2024-08-21T00:00:00+00:00 [queued]>
[2024-09-11T01:59:55.356+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.seccond_task scheduled__2024-08-21T00:00:00+00:00 [queued]>
[2024-09-11T01:59:55.357+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T01:59:55.395+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): seccond_task> on 2024-08-21 00:00:00+00:00
[2024-09-11T01:59:55.418+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=12726) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T01:59:55.414+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'seccond_task', 'scheduled__2024-08-21T00:00:00+00:00', '--job-id', '3435', '--raw', '--subdir', 'DAGS_FOLDER/7. our_first_dag.py', '--cfg-path', '/tmp/tmphmqj6r3l']
[2024-09-11T01:59:55.420+0000] {standard_task_runner.py:63} INFO - Started process 12742 to run task
[2024-09-11T01:59:55.420+0000] {standard_task_runner.py:91} INFO - Job 3435: Subtask seccond_task
[2024-09-11T01:59:55.529+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v4.seccond_task scheduled__2024-08-21T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T01:59:55.710+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='seccond_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-21T00:00:00+00:00'
[2024-09-11T01:59:55.712+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T01:59:55.749+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-11T01:59:55.755+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "hey, I am task2 and will be running after task 1"']
[2024-09-11T01:59:55.773+0000] {subprocess.py:86} INFO - Output:
[2024-09-11T01:59:55.777+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task 1
[2024-09-11T01:59:55.778+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-11T01:59:55.779+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T01:59:55.843+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=seccond_task, run_id=scheduled__2024-08-21T00:00:00+00:00, execution_date=20240821T000000, start_date=20240911T015955, end_date=20240911T015955
[2024-09-11T01:59:56.009+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T01:59:56.029+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
